vnext/docs/ProjectStructure.md

```diff
@@ -0,0 +1,100 @@
+# MSBuild Project Structure
+
+This is a description of each of the core projects, their purpose and output artifacts.<br/>
+Sample applications are not covered.
+
+## Alphabetical Index
+- [Chakra\Chakra.vcxitems](#Chakra)
+- [Desktop\React.Windows.Desktop.vcxproj](#React.Windows.Desktop)
+- **[Desktop.DLL\React.Windows.Desktop.DLL.vcxproj](#React.Windows.Desktop.DLL)**
+- [Desktop.IntegrationTests\React.Windows.Desktop.IntegrationTests.vcxproj](#React.Windows.Desktop.IntegrationTests)
+- [Desktop.UnitTests\React.Windows.Desktop.UnitTests.vcxproj](#React.Windows.Desktop.UnitTests)
+- [Folly\Folly.vcxproj](#Folly)
+- [FollyWin32\FollyWin32.vcxproj](#FollyWin32)
+- [IntegrationTestScripts\IntegrationTests.njsproj](#IntegrationTests-Node-Project)
+- [IntegrationTests\React.Windows.IntegrationTests.vcxproj](#React.Windows.IntegrationTests)
+- [ReactCommon\ReactCommon.vcxproj](#ReactCommon)
+- **[ReactUWP\ReactUWP.vcxproj](#ReactUWP)**
+- [ReactWindowsCore\ReactWindowsCore.vcxproj](#ReactWindowsCore)
+- [Shared\Shared.vcxitems](#Shared)
+- [Universal.IntegrationTests\React.Windows.Universal.IntegrationTests.vcxproj](#React.Windows.Universal.IntegrationTests)
+- [Universal.UnitTests\React.Windows.Universal.UnitTests.vcxproj](#React.Windows.Universal.UnitTests)
+
+## Common Projects
+
+### ReactWindowsCore
+*Static Library*<br/>
+Contains common functionality for both Desktop and Universal Windows variants.
+
+### Shared
+*Shared Items (no build artifact)*<br/>
+Holds sources common to both Windows variants, that require different build configuration
+(i.e. compiler flags, language standard).
+
+### ReactCommon
+*Static Library*<br/>
+React Native core, cross-platform C++ types and interfaces.<br/>
+Sources provided as part of the `react-native` Node dependency. Not part of this repository.<br/>
+See https://github.com/facebook/react-native/tree/v0.59.9/ReactCommon.
+
+### Folly
+*Static Library*<br/>
+Folly variant type system for JavaScript/C++ interoperability.<br/>
+Sources provided as part of the `react-native` Node dependency. Not part of this repository.
+
+### Chakra
+*Shared Items (no build artifact)*<br/>
+ChakraCore bridging layer. May use different compiler flags between Windows variants.
+
+### React.Windows.IntegrationTests
```

Comment:
>React.Windows.IntegrationTests [](start = 4, length = 30)

Might be worthwhile mentioning what projects the outputs of this project are flowing into.

Reply:
Can you elaborate on what you mean by "flowing into"?